2.2.1 Release Notes fixes (#4513)
mesosphere/dcos-docs-site: pages/dkp/kommander/2.2/release-notes/2.2.1/index.md

| Thanos                  | thanos                                         | 0.4.6   | - chart: 0.4.6<br>- thanos: 0.9.0                                                                                                       |
| Traefik                 | traefik                                        | 10.9.1  | - chart: 10.9.1<br>- traefik: 2.5.6                                                                                                     |
| Traefik ForwardAuth     | traefik-forward-auth                           | 0.3.6   | - chart: 0.3.6<br>- traefik-forward-auth: 3.1.0                                                                                         |
| Velero                  | velero                                         | 3.2.0   | - chart: 3.2.0<br>- velero: 1.5.2                                                                                                       |

## Known Issues

The following items are known issues with this release.

### Upgrade considerations

> Before attempting to upgrade an existing cluster to this release, check the 'kommander-vars' Configmap in the 'kommander' namespace for the following fields:

Reviewer: ```suggestion
Before attempting to upgrade an existing cluster to this release, check the `kommander-vars` Configmap in the `kommander` namespace for the following fields:
```

| Traefik ForwardAuth     | traefik-forward-auth                           | 0.3.6   | - chart: 0.3.6<br>- traefik-forward-auth: 3.1.0                                                                                         |
| Velero                  | velero                                         | 3.2.0   | - chart: 3.2.0<br>- velero: 1.5.2                                                                                                       |

## Known Issues

The following items are known issues with this release.

### Upgrade considerations

Before attempting to upgrade an existing cluster to this release, check the 'kommander-vars' Configmap in the 'kommander' namespace for the following fields:

- kommanderAppManagementImageTag
- kommanderAppManagementImageRepository
- kommanderChartsVersion

If any of the these fields are present, then there is a possibility the upgrade can fail.  If you encounter this situation, file a support ticket for advice on how to remediate the issue before attempting to continue the upgrade.

## Additional resources

> For more information about working with native Kubernetes, see the [Kubernetes documentation][kubernetes-doc].

Author: ```suggestion
For more information about working with native Kubernetes, see the [Kubernetes documentation][kubernetes-doc].

```

> Before attempting to upgrade an existing cluster to this release, please check the 'kommander-vars' Configmap in the 'kommander' namespace for the following fields:

Author: ```suggestion
Before attempting to upgrade an existing cluster to this release, check the 'kommander-vars' Configmap in the 'kommander' namespace for the following fields:
```
